setup.cfg

```diff
@@ -0,0 +1,55 @@
+[nosetests]
```

Comment:
We should have a conspicuous comment at the top indicating that this has been copied directly from `edx-platform` (unless, of course, you did something different here and it isn't a 1-for-1 copy)